makefile

```diff
@@ -2,41 +2,36 @@ GOTOOLS = github.com/golangci/golangci-lint/cmd/golangci-lint
 PACKAGES=$(shell go list ./...)
 INCLUDE = -I=. -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
 
+# Setup
+# See: LevelDB: https://github.com/jmhodges/levigo/blob/master/README.md
+# See: RocksDB: https://github.com/line/gorocksdb/blob/main/README.md
 CLEVELDB_DIR=$(shell pwd)/leveldb
-ROCKSDB_DIR=$(shell pwd)/rocksdb.build
+ROCKSDB_DIR=$(shell pwd)/rocksdb
 CGO_CFLAGS=-I$(CLEVELDB_DIR)/include -I$(ROCKSDB_DIR)/include
-CGO_LDFLAGS=-L$(CLEVELDB_DIR)/build -L$(ROCKSDB_DIR) -lleveldb -lrocksdb -lm -lstdc++ $(shell awk '/PLATFORM_LDFLAGS/ {sub("PLATFORM_LDFLAGS=", ""); print}' < $(ROCKSDB_DIR)/make_config.mk)
+CGO_LDFLAGS=-L$(CLEVELDB_DIR) -L$(ROCKSDB_DIR) -lleveldb -lrocksdb -lm -lstdc++ $(shell awk '/PLATFORM_LDFLAGS/ {sub("PLATFORM_LDFLAGS=", ""); print}' < $(ROCKSDB_DIR)/make_config.mk)
+
+DOCKER_NAME=tm-db-testing
+DOCKER_IMAGE=line/$(DOCKER_NAME)
 
 export GO111MODULE = on
 
 all: lint test
 
 ### go tests
 ## By default this will only test memdb & goleveldb
-test: cleveldb rocksdb.build
-	@echo "--> Running go test"
-	@CGO_CFLAGS="$(CGO_CFLAGS)" CGO_LDFLAGS="$(CGO_LDFLAGS)" \
-		go test $(PACKAGES) -tags memdb,goleveldb -v
-
-test-memdb:
-	@echo "--> Running go test"
-	@go test $(PACKAGES) -tags memdb -v
-
-test-goleveldb:
-	@echo "--> Running go test"
-	@go test $(PACKAGES) -tags goleveldb -v
-
-test-cleveldb:
+test:
 	@echo "--> Running go test"
-	@go test $(PACKAGES) -tags cleveldb -v
+	@go test $(PACKAGES) -v
 
-test-rocksdb:
+test-cleveldb: build-cleveldb
 	@echo "--> Running go test"
-	@go test $(PACKAGES) -tags rocksdb -v
+	@CGO_CFLAGS="$(CGO_CFLAGS)" CGO_LDFLAGS="$(CGO_LDFLAGS)" \
+	go test $(PACKAGES) -tags cleveldb -v
 
-test-rdb:
+test-rocksdb: build-rocksdb
 	@echo "--> Running go test"
-	@go test $(PACKAGES) -tags rocksdb -v
```

Comment:
Is the `test-rdb` no need?

Reply:
Yes, it is. Because it's the same as `test-rocksdb`.  There is no tag of `rdb`

```
@go test $(PACKAGES) -tags rocksdb -v
```

Reply:
OK, I understand. Thank you.